SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -119,17 +120,34 @@ SCDocHTMLRenderer {
 			baseDir = baseDir ++ "/..";
 		};
 
+		thisIsTheMainHelpFile = (doc.title=="Help") and:
+			{((thisProcess.platform.name===\windows) and: (folder=="Help")) or: {folder==""}};
+
+		stream
+		<< "<!doctype html>"
+		<< "<html lang='en'>"
+		<< "<head><title>";
+
+		if(thisIsTheMainHelpFile) {
```

Comment:
ok, but that sounds like it would take a lot more time than just refactoring a few blocks from the method.